README.md

```diff
@@ -50,6 +50,20 @@ Use the checklist below to ask questions about the candidate domain / service /
 * **Wardley Maps**: could the capability or service be outsourced to a SaaS Product or Commodity provider? Will it likely be outsource-able soon? If so, it's a candidate for splitting off as a separate service in preparation for possible outsourcing to a Product or Commodity provider. 
 * **Risk**: what is the cost of risk of splitting this capability or service? Could something go wrong?
 
+### Additional Considerations
+
+* **Skills Liquidity**: Team vs members of the team - considering skills liquidity (Chris Matts)
+* **Loose Couple**: Does this 'thing' make sense to independently migrate / deploy in the public cloud independant of related 'things' 
+* **Tight Coupling**: Do you have any dependencies on vendor / 3rd party software that prevents scaling if demand increases 
+* **Reuse**: Does an existing 'thing' / service already exist on public cloud that you could use as a commoditized service instead 
```

Comment:
Is this repetition of the Wardley Maps point above?

Reply:
yes good point, I had raised these ideas well before the DDD talk so will remove